index.html

```diff
@@ -13,8 +13,8 @@
 <body>
     <main>
         <h1>Numbers Game</h1>
-        <section id="splash" class="screen">
-            <button id="start" type="button">
+        <section id="splash_screen" class="screen">
```

Comment:
any particular reason for changing this?

Reply:
previous suggestion to make it slightly more semantic